uw-frame-components/portal/misc/partials/app-header.html

```diff
@@ -12,7 +13,24 @@ <h1 flex="60" class="app-title"><i class="fa {{::icon}}"></i> {{::title}}</h1>
         <div ng-include="optionTemplate"></div>
       </md-menu-content>
     </md-menu>
-    <!-- Add to home button -->
+
+    <!-- SINGLE-OPTION MENU -->
+    <div class="link-div" ng-if="optionTemplate && isSingleOption" hide-xs hide-sm>
+      <div ng-include="optionTemplate"></div>
+    </div>
+    <md-menu ng-if="optionTemplate && isSingleOption" hide-gt-sm>
+      <md-button aria-label="Open options menu" class="md-primary link-div" ng-click="$mdOpenMenu($event)">
+        <span layout="row" layout-align="center center"><i class='fa fa-gear fa-fw'></i></span>
```

Comment:
Mac OS X VoiceOver utility does not attempt to read the icon, and I'd expect other screen readers to behave the same because of the `aria-label` that @timlevett mentioned.